src/main/typescript/node_modules/@atomist/rug/operations/RugOperation.ts

```diff
@@ -102,9 +102,28 @@ class ReviewResult {
     this.comments.push(rc)
   }
 
-  static empty(note: string) {
+  static empty(note: string): ReviewResult {
```

Comment:
Given this is in RugOperation (and not a review specific file) shouldn't this be emptyReviewResult or some such?